buildchow/templates/defaults.yaml

```diff
@@ -4,35 +4,21 @@
     default-jdk: jdk1.8
     default-maven-version: maven-3.2.5
     default-remote: origin
-    jvm-memory: "-Xms512m -Xmx1024m -XX:PermSize=128m -XX:MaxPermSize=256m"
+    # since we use JDK 8 now, no need for these:  -XX:PermSize=128m -XX:MaxPermSize=256m"
+    jvm-memory: "-Xms512m -Xmx1536m"
     tests-folder: tests # for some it is all-tests
     component-responsible: ""
     scm-schedule: H H/8 * * * # every 8 hours starting a random hour. Removes need for each job to configure it (http://htmlpreview.github.io/?https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/hudson/triggers/TimerTrigger/help-spec.html)
-    download-cache:  "$JENKINS_HOME/static_build_env/jbds/download-cache"
-    maven-settings: "$JENKINS_HOME/../.m2/local-settings.xml" # need more reliable location
+    JENKINS_HOME: "/home/hudson"
```

Comment:
not following. why do you need it in jenkins job level ? $JENKINS_HOME is defined in jenkins already. JJB is better of *not* knowing the fixed path.